Documentation/tutorial/howto_create_custom_plugin.md

```diff
@@ -0,0 +1,160 @@
+How-to: Create Build Custom Document
+====================================
+
+In this topic, we will create a plug-in to build some simple rtf files to documents.
```

Comment:
1. provide a link to the definition of rtf extension?
2. describe what will be generated, what is the definition for `documents`? rtf files are also documents.